server/public_routes.go

```diff
@@ -34,6 +34,18 @@ func PublicRoutes(app *app.App) []*route.HandledRoute {
 		route.Get("/session/refresh").
 			SecuredWith(originSecurity).
 			Handle(handlers.GetSessionRefresh(app)),
+
+		route.Get("/totp").
```

Comment:
thoughts on `GET /totp/new`?

as it is currently, i might expect to get information about totp status.